### PR TITLE
Retire outdated app-creation doc

### PR DESCRIPTION
**Pull request by maorleger**
This commit retires the outdated app-creation doc and refers
users to the official Azure documentation.

It's possible that this doc is still referred to by various search
engines, blog posts, etc. so rather than deleting it I replaced it
with a tombstone doc directing folks to the right place.

### CODE REVIEW

**Comment by maorleger** [comment]
@ramya-rao-a  @daviwil @HarshaNalluru Sorry for the ping - for some reason GitHub isn't letting me add you as viewers so I couldn't get your attention any other way 😄

File: README.md [hunk 1]
```diff
@@ -7,10 +7,12 @@ Provides browser-based authentication for Azure resources. Designed for use with
 - node.js version > 6.x
 
 ## Getting Started
-Before using this library, it's necessary to create an Azure AD app in the portal. See [doc/app-creation.md](doc/app-creation.md) for instructions.
 
-Below is a basic sample with authenticates via Azure Active Directory. See the [samples](samples) for more detailed usage.
+Before using this library, it's necessary to create an Azure AD app in the portal. See [this documentation](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) for instructions.
```
Comment (ramya-rao-a) [inline README.md:11]: Is "creating an Azure AD app" the right term to use or "register the app" or "create the app registration"?

File: README.md [hunk 2]
```diff
@@ -7,10 +7,12 @@ Provides browser-based authentication for Azure resources. Designed for use with
...
+
+- Ensure that you have followed the steps under `MSAL.js 1.0 with Implicit Flow` as well.
```
Comment (ramya-rao-a) [inline README.md:13]: Can we add something to explain why though? https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-spa-app-registration#note-about-authorization-flows talks about how the default is different. Are we using msal.js v1 in ms-rest-browserauth or in adal (which is used in ms-rest-nodeauth) and therefore this is needed?

Comment (maorleger) [reply README.md]: I _think_ that ADAL.js uses the Implicit Grant Flow still. Only MSAL 2.0 supports the Auth Code Flow with PKCE as far as I can tell. I added a note about that here for posterity (but can elaborate if needed)

File: README.md (hunk 1 above)
Comment (maorleger) [reply README.md]: Yea I think creating an Azure App Registration is the better term. Updated!

**Comment by ramya-rao-a** [comment]
Looks good

@sadasant, this gives us an opportunity to clean up the readme for this package.
Is there anything you would like to add?

File: README.md [hunk 3]
```diff
@@ -7,10 +7,12 @@ Provides browser-based authentication for Azure resources. Designed for use with
 - node.js version > 6.x
 
 ## Getting Started
-Before using this library, it's necessary to create an Azure AD app in the portal. See [doc/app-creation.md](doc/app-creation.md) for instructions.
 
-Below is a basic sample with authenticates via Azure Active Directory. See the [samples](samples) for more detailed usage.
+Before using this library, it's necessary to create an Azure App Registration in the portal. See [this documentation](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) for instructions.
 
+- Please Ensure that you have followed the steps under `MSAL.js 1.0 with Implicit Flow` as well (this library uses ADAL.js which implements the Implicit Flow).
```
Comment (HarshaNalluru) [inline README.md:13]:
```suggestion
- Please ensure that you have followed the steps under `MSAL.js 1.0 with Implicit Flow` as well (this library uses ADAL.js which implements the Implicit Flow).
```

**Review by HarshaNalluru — APPROVED** [review]

**Comment by maorleger** [comment]
I'd love to merge this, but it still says "1 approving review is required with reviewers with write access". I figured Ramya or Harsha would have access but it looks like no - who does?

**Comment by HarshaNalluru** [comment]
@maorleger 
>  who does?

@ramya-rao-a and @daviwil should have the access

**Comment by ramya-rao-a** [comment]
I can approve, I just needed @sadasant to take a look and see if there is anything else he wants to add before we ship an update

**Review by sadasant — APPROVED** [review]
Loved it! Thanks.

**Comment by sadasant** [comment]
@ramya-rao-a I think the README is far as it is with these changes for now.

File: Changelog.md [hunk 4]
```diff
@@ -1,11 +1,18 @@
+## 1.0.2 - 2021-01-11
```
Comment (ramya-rao-a) [inline Changelog.md:1]:
```suggestion
## 1.0.2 - 2021-01-21
```

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**